plugins/obs-ffmpeg/obs-ffmpeg-source.c

```diff
@@ -288,6 +300,44 @@ static void ffmpeg_source_open(struct ffmpeg_source *s)
 	}
 }
 
+static void ffmpeg_source_start(struct ffmpeg_source *s)
+{
+	if (!s->media_valid)
+		ffmpeg_source_open(s);
+
+	if (!s->media_valid)
+		return;
+
+	mp_media_play(&s->media, s->is_looping);
+	if (s->is_local_file)
+		obs_source_show_preloaded_video(s->source);
```

Comment:
Add a blank line between this line and the next line to separate `if` contents from the code that comes after it.